app/views/jobseekers/job_applications/build/personal_statement.html.slim

```diff
@@ -17,10 +17,23 @@
       - else
         = t(".banner_no_documents_html", job_link: open_in_new_tab_link_to("‘#{job_application.vacancy.job_title}’", job_path(job_application.vacancy)))
 
+    .help-guide-container--mobile
+      h3.govuk-heading-m = t(".personal_statement_guide.title")
+      p.govuk-body = t(".personal_statement_guide.description_mobile")
+      = govuk_link_to(t(".personal_statement_guide.link_text"),
+                                                      post_path(section: "jobseeker-guides", post_name: "how-to-write-teacher-personal-statement"),
+                                                      class: "govuk-link--no-visited-state")
+
     - if job_application.vacancy.personal_statement_guidance.present?
       h3.govuk-heading-s = t(".additional_instructions")
       = govuk_inset_text text: job_application.vacancy.personal_statement_guidance
 
+  - if current_jobseeker.job_applications.not_draft.none?
+    .govuk-grid-column-one-third
+      = render "steps"
```

Comment:
Just checking that we want the steps above the "How to write a teacher personal statement" card 

![Screenshot 2022-03-21 at 18 04 19](https://user-images.githubusercontent.com/30624173/159336310-c9d03e7d-729e-41f2-94e1-b1e58672d783.png)